### PR TITLE
add metrics

### PR DESCRIPTION
**Pull request by modship**
close : #4224

### CODE REVIEW

File: massa-metrics/src/lib.rs [hunk 1]
```diff
@@ -306,23 +431,58 @@ impl MassaMetrics {
...
             }
 
-            MassaSurvey::run(
-                tick_delay,
-                active_in_connections.clone(),
-                active_out_connections.clone(),
-                peernet_total_bytes_sent.clone(),
-                peernet_total_bytes_receive.clone(),
-            );
+            // MassaSurvey::run(
```
Comment (modship) [inline massa-metrics/src/lib.rs:456]: to remove

File: massa-protocol-worker/src/connectivity.rs [hunk 2]
```diff
@@ -240,6 +241,9 @@ pub(crate) fn start_connectivity_thread(
                         massa_metrics.set_active_connections(active_conn.get_nb_in_connections(), active_conn.get_nb_out_connections());
                         let peers_map = active_conn.get_peers_connections_bandwidth();
                         massa_metrics.update_peers_tx_rx(peers_map);
+                        let peer_db_read = peer_db.read();
+                        massa_metrics.set_known_peers(peer_db_read.peers.len());
+                        massa_metrics.set_banned_peers(peer_db_read.peers.iter().filter(|(_id, info)| info.state == PeerState::Banned).count());
```
Comment (Leo-Besancon) [inline massa-protocol-worker/src/connectivity.rs:246]: You should have 
 ```rust
peer_db.get_banned_peer_count()
```
accessible here

**Review by Leo-Besancon — APPROVED** [review]
Looks good to me

File: massa-protocol-worker/src/connectivity.rs (hunk 2 above)
Comment (modship) [reply massa-protocol-worker/src/connectivity.rs]: yes !!! thx, 
Done

**Pull request merged**